### DISCUSSION TITLE
21.02的upnp不工作

### DISCUSSION THREAD

**Discussion by lee29** [immortalwrt/immortalwrt#747] [Q&A]
RT
内核5.4
libminiupnpc libupnp miniupnpd都安装了

用lede的没问题,希望比对修复一哈

**Comment by 1715173329** [comment]
1. 如果你用的某些程序仍在使用老旧的 upnp 套件，那么无法支持。
2. 尽可能使用端口转发。
3. No log no solution.

**Comment by Jecvay** [comment]
我这里 21.02 miniupnpd 工作正常
